docs/kagi/ai/assistant.md

```diff
@@ -166,6 +166,12 @@ You can quickly access Assistant from Kagi Search by using the following [bangs]
 
 You can also access the **Code** Custom Assistant with the `!code` bang.
 
+## URL Parameter for Model Selection
+
+You can specify a particular model in the Assistant's URL by including a `profile` parameter.
+`https://kagi.com/assistant?profile=gpt_4o&q=%s`
+The available model names include: `gpt_4o` `gpt_4o_mini` `gpt_4` `gpt_4_turbo` `claude_3_haiku` `claude_3_opus` `claude_3_sonnet` `code` `gemini_pro` `llama-3-405b` `mistral_pixtral` `mistral_large`
```

Comment:
`code` is one of the default assistants, not a model, though that's semantics for this.